src/main/kotlin/commands/IssueCommand.kt

```diff
@@ -134,21 +136,12 @@ object IssueCommand : Command("issue") {
                         field("Commits", pullRequest.commits, false)
                         field("Changed Files", pullRequest.changed_files, false)
 
-                        url = pullRequest.url
+                        url = pullRequest.html_url
                     }
                 }
             }
         } catch (e: Exception) {
-            message.channel.send {
-                embed {
-                    title = "Error"
-                    description =
-                        "Something went wrong when trying to execute this command! Does the user / repo / issue exist?"
-                    field("Stacktrace", "```$e```", false)
-                    e.printStackTrace()
-                    color = Colors.error
-                }
-            }
+            message.stackTrace(e)
```

Comment:
See above. You should still have a user friendly title. Maybe add the title option to `stackTrace()`?